In [1]:
library(tidyverse)
library(mosaicData)

-- Attaching packages ------------------------------------------------------------------------------- tidyverse 1.3.0 --

v ggplot2 3.3.3     v purrr   0.3.4
v tibble  3.0.4     v dplyr   1.0.2
v tidyr   1.1.2     v stringr 1.4.0
v readr   1.4.0     v forcats 0.5.0

-- Conflicts ---------------------------------------------------------------------------------- tidyverse_conflicts() --
x dplyr::filter() masks stats::filter()
x dplyr::lag()    masks stats::lag()



In [2]:
pbinom(
    5,
    7,
    0.65
)

[1] 0.7662014

In [3]:
pnorm(
    5,
    mean=4,
    sd=2.2,
    lower.tail=FALSE
)

[1] 0.3247181

In [4]:
ppois(
    6,
    4,
    lower.tail=FALSE
)

[1] 0.110674

In [5]:
str(CPS85)

'data.frame':	534 obs. of  11 variables:
 $ wage    : num  9 5.5 3.8 10.5 15 9 9.57 15 11 5 ...
 $ educ    : int  10 12 12 12 12 16 12 14 8 12 ...
 $ race    : Factor w/ 2 levels "NW","W": 2 2 2 2 2 2 2 2 2 2 ...
 $ sex     : Factor w/ 2 levels "F","M": 2 2 1 1 2 1 1 2 2 1 ...
 $ hispanic: Factor w/ 2 levels "Hisp","NH": 2 2 2 2 2 2 2 2 2 2 ...
 $ south   : Factor w/ 2 levels "NS","S": 1 1 1 1 1 1 1 1 1 1 ...
 $ married : Factor w/ 2 levels "Married","Single": 1 1 2 1 1 1 1 2 1 1 ...
 $ exper   : int  27 20 4 29 40 27 5 22 42 14 ...
 $ union   : Factor w/ 2 levels "Not","Union": 1 1 1 1 2 1 2 1 1 1 ...
 $ age     : int  43 38 22 47 58 49 23 42 56 32 ...
 $ sector  : Factor w/ 8 levels "clerical","const",..: 2 7 7 1 2 1 8 7 4 7 ...


In [6]:
table(CPS85$sex, CPS85$married)

   
    Married Single
  F     162     83
  M     188    101

In [7]:
married_cont_table <- table(CPS85$sex, CPS85$married)%>% prop.table()
married_cont_table

   
      Married    Single
  F 0.3033708 0.1554307
  M 0.3520599 0.1891386

In [8]:
married_given_F <- married_cont_table[1,1]/(married_cont_table[1,1]+married_cont_table[1,2])

In [9]:
married_given_F

[1] 0.6612245

The probability of a person being married in 1985 given that they are female is around 66.12%

In [10]:
movies <- read.csv("Movies.csv")
movies

Title                                                          Year
1   Star Wars Ep. VII: The Force Awakens                           2015
2   Avatar                                                         2009
3   Titanic                                                        1997
4   Jurassic World                                                 2015
5   The Avengers                                                   2012
6   Star Wars Ep. VIII: The Last Jedi                              2017
7   The Dark Knight                                                2008
8   Rogue One: A Star Wars Story                                   2016
9   Beauty and the Beast                                           2017
10  Finding Dory                                                   2016
11  Star Wars Ep. I: The Phantom Menace                            1999
12  Star Wars Ep. IV: A New Hope                                   1977
13  Avengers: Age of Ultron                                        2015
14  The Dark Knight Rises                                          2012
15  Shrek 2                                                        2004
16  ET: The Extra-Terrestrial                                      1982
17  The Hunger Games: Catching Fire                                2013
18  Pirates of the Caribbean: Dead ManÕs Chest                     2006
19  The Lion King                                                  1994
20  Toy Story 3                                                    2010
21  Wonder Woman                                                   2017
22  Iron Man 3                                                     2013
23  Captain America: Civil War                                     2016
24  The Hunger Games                                               2012
25  Spider-Man                                                     2002
26  Transformers: Revenge of the Fallen                            2009
27  Frozen                                                         2013
28  Jurassic Park                                                  1993
29  Guardians of the Galaxy Vol 2                                  2017
30  Harry Potter and the Deathly Hallows: Part 1                   2011
... ...                                                            ... 
71  The Hobbit: An Unexpected Journey                              2012
72  Harry Potter and the Half-Blood Prince                         2009
73  The Twilight Saga: Eclipse                                     2010
74  The Twilight Saga: New Moon                                    2009
75  Harry Potter and the Deathly Hallows:Part 2                    2010
76  The Sixth Sense                                                1999
77  Up                                                             2009
78  Inception                                                      2010
79  The Twilight Saga: Breaking Dawn, Part 2                       2012
80  Harry Potter and the Order of the Phoenix                      2007
81  The Chronicles of Narnia: The Lion, The Witch, and The Wadrobe 2005
82  Man of Steel                                                   2013
83  Star Wars Ep. V: The Empire Strikes Back                       1980
84  Harry Potter and the Goblet of Fire                            2005
85  Monsters, Inc.                                                 2001
86  Home Alone                                                     1990
87  The Hunger Games: Mockingjay - Part 2                          2015
88  The Matrix Reloaded                                            2003
89  The Twilight Saga: Breaking Dawn, Part 1                       2011
90  Meet the Fockers                                               2004
91  The Hangover                                                   2009
92  Gravity                                                        2013
93  Sing                                                           2016
94  Monsters University                     

In [11]:
mov <- movies %>% 
select(Title, Year, Distributor, Type, Budget, IMDB.Rating, Opening.Theaters, Opening.Revenue) %>%
    mutate(
        profit = Opening.Revenue - Budget,
        disney = Distributor == "Walt Disney"
    )

In [12]:
mov

Title                                                          Year
1   Star Wars Ep. VII: The Force Awakens                           2015
2   Avatar                                                         2009
3   Titanic                                                        1997
4   Jurassic World                                                 2015
5   The Avengers                                                   2012
6   Star Wars Ep. VIII: The Last Jedi                              2017
7   The Dark Knight                                                2008
8   Rogue One: A Star Wars Story                                   2016
9   Beauty and the Beast                                           2017
10  Finding Dory                                                   2016
11  Star Wars Ep. I: The Phantom Menace                            1999
12  Star Wars Ep. IV: A New Hope                                   1977
13  Avengers: Age of Ultron                                        2015
14  The Dark Knight Rises                                          2012
15  Shrek 2                                                        2004
16  ET: The Extra-Terrestrial                                      1982
17  The Hunger Games: Catching Fire                                2013
18  Pirates of the Caribbean: Dead ManÕs Chest                     2006
19  The Lion King                                                  1994
20  Toy Story 3                                                    2010
21  Wonder Woman                                                   2017
22  Iron Man 3                                                     2013
23  Captain America: Civil War                                     2016
24  The Hunger Games                                               2012
25  Spider-Man                                                     2002
26  Transformers: Revenge of the Fallen                            2009
27  Frozen                                                         2013
28  Jurassic Park                                                  1993
29  Guardians of the Galaxy Vol 2                                  2017
30  Harry Potter and the Deathly Hallows: Part 1                   2011
... ...                                                            ... 
71  The Hobbit: An Unexpected Journey                              2012
72  Harry Potter and the Half-Blood Prince                         2009
73  The Twilight Saga: Eclipse                                     2010
74  The Twilight Saga: New Moon                                    2009
75  Harry Potter and the Deathly Hallows:Part 2                    2010
76  The Sixth Sense                                                1999
77  Up                                                             2009
78  Inception                                                      2010
79  The Twilight Saga: Breaking Dawn, Part 2                       2012
80  Harry Potter and the Order of the Phoenix                      2007
81  The Chronicles of Narnia: The Lion, The Witch, and The Wadrobe 2005
82  Man of Steel                                                   2013
83  Star Wars Ep. V: The Empire Strikes Back                       1980
84  Harry Potter and the Goblet of Fire                            2005
85  Monsters, Inc.                                                 2001
86  Home Alone                                                     1990
87  The Hunger Games: Mockingjay - Part 2                          2015
88  The Matrix Reloaded                                            2003
89  The Twilight Saga: Breaking Dawn, Part 1                       2011
90  Meet the Fockers                                               2004
91  The Hangover                                                   2009
92  Gravity                                                        2013
93  Sing                                                           2016
94  Monsters University                     

In [13]:
movie <- mov %>%
  mutate(
    profit = cut(profit,
      breaks = c(-Inf,0,Inf),
      labels = c("no profit", "profit")
) )

In [14]:
opening_profit <- table(movie$profit, movie$disney)%>% prop.table()
opening_profit

           
            FALSE TRUE
  no profit  0.54 0.27
  profit     0.17 0.02

In [24]:
profit_given_disney <- opening_profit[2,2]/(opening_profit[1,2]+opening_profit[2,2])

In [25]:
profit_given_disney

[1] 0.06896552

The conditional probability of making a profit during opening given that it is a Disney film is roughly 6.89%

In [16]:
chisq.test(movie$profit, movie$disney)


	Pearson's Chi-squared test with Yates' continuity correction

data:  movie$profit and movie$disney
X-squared = 2.8592, df = 1, p-value = 0.09086


My interpretation of the results of the previous part is that since the p-value is not small (less than 5%), I can't say for certain whether the two variables are independent or not. So I know neither if the variables are independent or not. However the p value is certainly closer to 5% so I would bet that the two variables are independent of each other.